src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React, { type ComponentType } from 'react';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { StagesAndEventsWidget } from '../../../Widgets';
+import type { Props, PlainProps } from './EnrollmentPageDefault.types';
+
+const getStyles = ({ typography }) => ({
+    columns: {
+        display: 'flex',
+    },
+    leftColumn: {
+        flexGrow: 3,
+        flexShrink: 1,
+        width: 872,
+    },
+    rightColumn: {
+        flexGrow: 1,
+        flexShrink: 1,
+        paddingLeft: 16,
```

Comment:
Hey, 
I see that you use `spacersNum.dp16` in another part of this PR to set the paddings and margins. Can you use the same pattern in this file? 
Thanks!

Reply:
Excellent point. Will fix, let me know if I missed this in other places.